client/my-sites/checkout/composite-checkout/components/additional-terms-of-service-in-cart.tsx

```diff
@@ -74,6 +74,40 @@ function getMessageForTermsOfServiceRecord(
 				args.subscription_expiry_date &&
 				args.subscription_auto_renew_date
 			) {
+				if (
+					args.is_renewal_price_prorated &&
+					( locale === 'en' ||
+						i18nCalypso.hasTranslation(
+							'The promotional period for your subscription lasts from %(startDate)s to %(endDate)s. On %(renewalDate)s we will charge your payment method (PAYPAL %(email)s) for %(renewalPrice)s. All subsequent renewals will be charged for the regular subscription price of %(regularPrice)s. You will receive at least one email notice %(numberOfDays)d days before being billed and can {{updatePaymentMethodLink}}update your payment method{{/updatePaymentMethodLink}} or {{manageSubscriptionlink}}manage your subscription {{/manageSubscriptionlink}} at any time.'
```

Comment:
Minor, but there's a space after `manage your subscription` which is included inside the link and it looks a little weird in the UI to have that space be underlined.

Reply:
Thanks, I noticed that the extra space is also in the strings that were already translated. How should I proceed? Should I fix them as well?

Reply:
😬 oh, that's not ideal. I don't know the best choice here. It would be nice to fix it but it does mean we'd need more translations. I wonder if it's possible to have someone edit the translations to remove it somehow? We should maybe ask a localization expert. Anyway, this is not a blocker so do whatever you think best.

Reply:
@Automattic/i18n can confirm, but I believe there's a certain degree of fuzzy matching in our translation tools. If you flag this string with them, they might be able to make sure it doesn't need a re-translation.